src/tsolvers/lrasolver/Tableau.h

```diff
@@ -12,6 +12,7 @@
 
 #include <unordered_set>
```

Comment:
Are these still used somewhere?  Maybe remove them everywhere and steer clear of them to avoid similar problems?

Reply:
Yes, you are right, it should not be used anywhere else, so try to remote it if it works.